js/components/tooltip.js

```diff
@@ -0,0 +1,114 @@
+var React = require('react');
+var deepmerge = require('deepmerge');
+
+function getContainerStyle(bottom, visible, responsivenessMultiplier, alignment) {
+    var verticalAlignment = 80 * responsivenessMultiplier;
+    var alignmentStyle = {
+        left: {
+            left: 0,
+            transform: 'translate(0,' + verticalAlignment + '%)'
+        },
+        center: {
+            left: '50%',
+            transform: 'translate(-50%,' + verticalAlignment + '%)'
+        },
+        right: {
+            right: 0,
+            transform: 'translate(0,' + verticalAlignment + '%)'
+        }
+    };
+
+    var style = {
+        position: 'absolute',
+        color: 'white',
+        fontFamily: '"Roboto Condensed", sans-serif',
+        opacity: visible ? '0.75' : '0',
+        fontWeight: 'normal',
+        bottom: bottom,
+        //display: visible ? 'block' : 'none',
+        transition:'1s',
+        visibility: visible ? 'visible' : 'hidden',
+    }
+
+    return deepmerge(alignmentStyle[alignment], style);
+
+}
+
+function getBoxStyle(responsivenessMultiplier) {
+    return {
+        borderRadius: '3px',
+        fontSize: '15px',
+        textShadow: 'none',
+        background: 'black',
+        paddingTop: 8 * responsivenessMultiplier,
```

Comment:
Do we need to do any kind of rounding so it's always a whole number?  Does it create blurring if it's a decimal? And does that matter?